src/scripts/main.js

```diff
@@ -1,3 +1,72 @@
 'use strict';
 
-// write code here
+const addRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const addColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+const bodyTable = document.querySelector('tbody');
+
+const min = 2;
+const max = 10;
+
+addRow.addEventListener('click', () => {
+  if (bodyTable.children.length <= max) {
```

Comment:
According to previous recommendation - we can create a separated variable for `bodyTable.children.length`
`const length = bodyTable.children.length`
```suggestion
  if (length <= max) {
```

Reply:
this comment didn't fix